spec/factories/tasks.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :task do
+    sequence(:title) { |n| "rspec_test_#{n}" }
+    content { "content" }
+    status   { 'todo' }
```

Comment:
enumの形式で記載するようにしましょう
```suggestion
    status   { :todo }
```

Reply:
60d43adf5fe1847754d7be77c16f6b652ae3c6e1 で修正しました